scripts/update-agent-context.sh

```
@@ -0,0 +1,234 @@
+        commands_section = re.search(r'## Commands\n(.*?)\n\n', content, re.DOTALL)
+    
+    if commands_section:
+        new_commands = commands_section.group(1)
+        if "Python" in "$NEW_LANG":
+            new_commands += "\ncd src && pytest && ruff check ."
+        elif "Rust" in "$NEW_LANG":
+            new_commands += "\ncargo test && cargo clippy"
+        elif "JavaScript" in "$NEW_LANG" or "TypeScript" in "$NEW_LANG":
+            new_commands += "\nnpm test && npm run lint"
+        
+        if "```bash" in content:
```

Shellcheck (0.10.0)

[error] Couldn't parse this backtick expansion. Fix to allow more checks.

(SC1073)